packages/core/src/App/Containers/Layout/header/default-mobile-links.tsx

```diff
@@ -18,9 +18,11 @@ const DefaultMobileLinks = React.memo(({ handleClickCashier }: TDefaultMobileLin
     const { is_next_wallet_enabled } = useFeatureFlags();
     return (
         <React.Fragment>
-            <div className='traders-hub-header__menu-right--items--onboarding'>
-                <TradersHubOnboarding />
-            </div>
+            {is_next_wallet_enabled ? (
+                <div className='traders-hub-header__menu-right--items--onboarding'>
+                    <TradersHubOnboarding />
+                </div>
+            ) : null}
```

Comment:
Can't we use this 
Because it works the same.
If `is_next_wallet_enabled` is truthy, the right-hand side is evaluated, and the JSX block is rendered. 
This is a concise way of conditionally rendering elements in React.
```suggestion
           {is_next_wallet_enabled && (
                <div className='traders-hub-header__menu-right--items--onboarding'>
                    <TradersHubOnboarding />
                </div>
            )}
```